src/sentry/integrations/vsts/integration.py

```diff
@@ -81,6 +36,31 @@ def get_client(self):
             raise ValueError('Identity missing access token')
         return VstsApiClient(access_token)
 
+    def get_project_config(self):
+        client = self.get_client()
+        projects = client.get_projects(self.model.metadata['domain_name'])
+        project_choices = [(project['id'], project['name']) for project in projects['value']]
+        default_project = self.org_integration.config.get('default_project')
+
+        # TODO(LB): This is only saving the id. That might work... but really I'd
+        # want to save the name. Help? Maybe name and id?
+        return [
+            {
+                'name': 'default_project',
+                'type': 'choice',
+                'allowEmpty': True,
+                'required': True,
+                'choices': project_choices,
+                'initial': (default_project['id'], default_project['name']) if default_project is not None else ('', ''),
+                # TODO(LB): Tried using ugettext_lazy but got <django.utils.functional.__proxy__ object at 0x107fb5110> is not JSON serializable
```

Comment:
Never could figure out how to get ugettext_lazy to serialize. It seems to only fail right after the integration installation process finishes. I pulled it out, but would like help knowing why it doesn't work.

Reply:
i wonder if it has to do with using the `_lazy` version. can you try `ugettext`? didn't really have time to dig into the differences, but here are the docs: https://docs.djangoproject.com/en/1.7/topics/i18n/translation/

Reply:
We probably avoided this on JIRA since we don't actually install the integration through the normal flow. You probably do want this to not be lazy, we'd probably want the JIRA one to not be lazy either.